src/features/Chat/components/ActiveWindow/index.tsx

```diff
@@ -0,0 +1,234 @@
+    sendMessage({ userId, message });
+    setText('');
+  };
+
+  const isLoading = isLoadingMessages || isMessageSendLoading;
+
+  const getProfileIcon = (status: ChatFolder) => {
+    const icons = {
+      ok: <ProfileIcon color="purpleDark" />,
+      archived: <img src={ArchivedIcon} />,
+      banned: <img src={BlockedIcon} />,
+    };
+
+    return icons[status];
+  };
+
+  return (
+    chat && (
+      <ActiveChatContainer>
+        <HeaderBar>
+          <ProfileInfo>
+            {getProfileIcon(chat.status)}
+            <MentorName variant="h2">{chat.displayName}</MentorName>
+            <MentorBio variant="p">{chat.status}</MentorBio>
+          </ProfileInfo>
+          {showSearch ? (
+            <SearchBar>
+              <SearchInput
+                variant="iconInput"
+                color={searchValue ? 'blueDark' : 'greyFaded'}
+                leftIcon={{
+                  sizeInPx: searchInputIconSize,
+                  variant: 'search',
+                }}
+                rightButton={{
+                  onClick: () => setShowSearch(false),
+                  sizeInPx: closeInputIconSize,
+                  variant: 'closeWithBackground',
+                }}
+                onChange={setSearchValue}
+                placeholder={t('header.search')}
+                value={searchValue}
+              />
+            </SearchBar>
+          ) : (
+            <Buttons>
+              <IconButton
+                variant="search"
+                sizeInPx={24}
+                onClick={() => setShowSearch(true)}
+              />
+              {chat.status === 'ok' ? (
+                <>
+                  <StatusButton
+                    onClick={() => updateStatus('archived')}
+                    icon="archive"
+                    text={t('header.archive')}
+                  />
+                  <StatusButton
+                    onClick={() => updateStatus('banned')}
+                    icon="block"
+                    text={t('header.block')}
+                  />
+                </>
+              ) : (
+                <StatusButton
+                  onClick={() => updateStatus('ok')}
+                  icon="return"
+                  text={t('header.restore')}
+                />
+              )}
+            </Buttons>
+          )}
+        </HeaderBar>
+        <MessageList
+          messageList={chat.messages}
+          buddyId={chat.buddyId}
+          status={chat.status}
+          isLoading={isLoading}
+        />
+        <MessageField>
+          <Input
+            variant="textarea"
+            color={text ? 'blueDark' : 'greyFaded'}
+            onChange={setText}
+            placeholder={t('input.placeholder')}
+            value={text}
+          />
+          <SendButton
+            variant="send"
+            sizeInPx={46}
+            onClick={() => handleMessageSend(chat.buddyId, text)}
+          />
+        </MessageField>
```

Comment:
This could be a nice abstrated component, because it contains the messagefield and button